### ISSUE TITLE
Trigger profile.deleteCache(uid) from subhub messaging?

### ISSUE THREAD

**Issue by lmorchard** [mozilla/fxa#2799]
Noticed that `lib/subhub/updates.js` in auth-server just calls `db.createAccountSubscription` and `db.deleteAccountSubscription`. 

Meanwhile, the API route that handles creating a subscription [does a few more things](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/subscriptions.js#L153-L166) to notify services and invalidate the profile-server cache:

```js
        await db.createAccountSubscription({
          uid,
          subscriptionId,
          productId,
          createdAt: Date.now(),
        });

        const devices = await request.app.devices;
        await push.notifyProfileUpdated(uid, devices);
        log.notifyAttachedServices('profileDataChanged', request, {
          uid,
          email,
        });
        await profile.deleteCache(uid);
```

Should we add these things to the subhub message processing? I'm wondering if maybe there will be a lag in subscription deletion being reflected in the userinfo endpoint data from profile-server - i.e. subscriptions will be reported as active longer than they should after automated cancellation

**Comment by bbangert** [comment]
Yes, I think they should be added to the subhub message handling.

**Comment by bbangert** [comment]
Fixed by #2895.
